corehq/apps/locations/views.py

```diff
@@ -474,9 +474,9 @@ def users_form(self):
         return form
 
     @property
-    def all_products(self):
+    def active_products(self):
         return [(p.product_id, p.name)
-                for p in SQLProduct.by_domain(self.domain)]
+                for p in SQLProduct.objects.filter(domain=self.domain, is_archived=False).all()]
```

Comment:
Agreed.  That should have been made the default as part of introducing the archival workflow.  I was looking at a similar area recently and made a note to switch this over when I get the chance.